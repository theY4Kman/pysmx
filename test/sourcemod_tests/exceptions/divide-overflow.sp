// returnCode: 1
public main()
{
  int a = 0x80000000;
  int b = -1;
  return a / b;
}
