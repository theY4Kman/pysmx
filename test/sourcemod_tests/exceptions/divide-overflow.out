Exception thrown: Integer overflow
  [0] divide-overflow.sp::main, line 6
